src/Listeners/RabbitMQTriggerObserver.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using RabbitMQ.Client.Events;
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Listeners
+{
+    internal class RabbitMQTriggerObserver
+    {
+        private readonly ITriggeredFunctionExecutor executor;
+        private readonly EventingBasicConsumer consumer;
+
+        public RabbitMQTriggerObserver(ITriggeredFunctionExecutor executor, EventingBasicConsumer consumer)
+        {
+            this.executor = executor;
+            this.consumer = consumer;
+        }
+
+        public Task ProcessChangesAsync(CancellationToken cancellationToken)
+        {
+            this.consumer.Received += (model, ea) =>
+            {
+                var body = ea.Body;
+                var message = Encoding.UTF8.GetString(body);
+                Console.WriteLine(" [x] Received {0}", message);

```

Comment:
do not log message as this is user data

Reply:
Should I actually just remove this file since it's included in the next PR? It's part of the trigger logic